examples/testsuites/twonode.py

```diff
@@ -0,0 +1,51 @@
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.testsuite import simple_requirement
+from lisa.tools import Ntttcp, Uname
+
+
+@TestSuiteMetadata(
+    area="demo",
+    category="demo",
+    description="""
+    this is an example test suite.
+    It helps to understand how test cases works on multiple nodes
+    """,
+    tags=["demo", "twonode"],
+    requirement=simple_requirement(min_count=2),
+)
+class NtttcpDemo(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        this test case send and receive data by ntttcp
+        """,
+        priority=1,
+    )
+    def os_info(self) -> None:
+        self._log.info(f"node count: {len(self.environment.nodes)}")
+
+        for index in range(len(self.environment.nodes)):
+            node = self.environment.nodes[index]
+            uname = node.tools[Uname]
+            release, version, hardware, os = uname.get_linux_information()
```

Comment:
Maybe the Uname tool could return a NamedTuple instead?  It's very easy to swap these return values.  (And, if we ever need to add more of them in the future, we don't need to patch every place that calls `get_linux_information()`.)

Reply:
Have a try on namedtuple, but it looks a dataclass is better for strong type checks. fixed this style.